### PR TITLE
Readme suggestions and comments

### PR DESCRIPTION
**Pull request by hud10837**
Hey @TADraeseke , these are the fixes I mentioned that I was working on if you could look at these before looking at the big branch. Thanks!

### CODE REVIEW

File: java/buffer/README.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ Creating buffers is a core concept in GIS proximity analysis that allows you to
 
 ## How it works
 
-1. The map `Point` for a tap on the display is captured.
-2. The static method `GeometryEngine.buffer(Point, double)` is called to create a planar buffer polygon from the map location and distance.
-3. Another static method, `GeometryEngine.bufferGeodetic` is called to create a geodesic buffer polygon using the same inputs.
+1. Capture the map `Point` for a tap on the display.
+2. Call the static method `GeometryEngine.buffer(Point, double)` to create a planar buffer polygon from the map location and distance.
```
Comment (TADraeseke) [inline java/buffer/README.md]:
```suggestion
2. Call `GeometryEngine.buffer(Point, double)` to create a planar buffer polygon from the map location and distance.
```

File: java/buffer/README.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ Creating buffers is a core concept in GIS proximity analysis that allows you to
...
+3. Call another static method, `GeometryEngine.bufferGeodetic`, to create a geodesic buffer polygon using the same inputs.
```
Comment (TADraeseke) [inline java/buffer/README.md]:
```suggestion
3. Call `GeometryEngine.bufferGeodetic(...)`, to create a geodesic buffer polygon using the same inputs.
```

File: kotlin/choose-camera-controller/README.md [hunk 3]
```diff
@@ -14,17 +14,17 @@ The application loads with the default globe camera controller. To rotate and fi
 
 ## How it works
 
-1.  Create an instance of a class extending `CameraController`: `GlobeCameraController`, `OrbitLocationCameraController`, `OrbitGeoElementCameraController`.
-1.  Set the scene view's camera controller with `sceneView.cameraController = CameraController`.
+1. Create an instance of a class extending `CameraController`: `GlobeCameraController`, `OrbitLocationCameraController`, `OrbitGeoElementCameraController`.
+1. Set the scene view's camera controller with `sceneView.cameraController = CameraController`.
```
Comment (TADraeseke) [inline kotlin/choose-camera-controller/README.md]:
```suggestion
2. Set the scene view's camera controller with `sceneView.cameraController = CameraController`.
```

**Comment by hud10837** [comment]
Thanks @TADraeseke merging now :)

**Pull request merged**